src/backends/perf_pt/mod.rs

```diff
@@ -80,7 +80,9 @@ impl Error for LibIPTError {
 
 #[repr(C)]
 #[allow(dead_code)] // Only C constructs these.
+#[derive(PartialEq)]
 enum PerfPTCErrorType {
```

Comment:
If we want to make this Rust-y it might be better to use `Kind` rather than `Type`.

Reply:
Sure, are you happy for me to update the name on the C side too?

Reply:
Yep.